debian/changelog

```diff
@@ -1,4 +1,4 @@
-xivo-nginx (16.13~~) xivo-dev; urgency=medium
+wazo-nginx (16.13~~) xivo-dev; urgency=medium
```

Comment:
Could you bump the changelog version with an entry "renaming to wazo-nginx"?

Reply:
ok